### PR TITLE
Creator Statement

### PR DESCRIPTION
**Pull request by andorp**

### CODE REVIEW

**Comment by andorp** [comment]
Note: only the add creator statement button is displayed for the creator of an idea. The creator statement is displayed to everyone.

File: src/Frontend.hs [hunk 1]
```diff
@@ -234,6 +234,8 @@ type IdeaApi
   :<|> Idea ::> Comment ::> CommentApi
        -- jury an idea
   :<|> Idea ::> IdeaJuryResultType ::> FormHandler JudgeIdea
+       -- add creator statement
+  :<|> Idea ::> "statement" :> FormHandler CreatorStatement
```
Comment (fisx) [inline src/Frontend.hs]: `"creatorstatement"`?  (this is ok, but i'd prefer something more explicit.)

File: src/Frontend/Page/Idea.hs [hunk 2]
```diff
@@ -157,6 +164,22 @@ instance ToHtml ViewIdea where
...
 
+            when (CanAddCreatorStatement `elem` caps) $ do
+                let newStatement = isNothing $ creatorStatementOfIdea idea
+                div_ [class_ "creator-statement-button"] $ do
+                    button_ [ class_ "btn-cta m-valid"
+                            , onclick_ $ U.creatorStatement idea
+                            ] $ do
+                        i_ [class_ "icon-check"] nil
+                        if newStatement
+                            then "ADD CREATOR STATEMENT"
```
Comment (fisx) [inline src/Frontend/Page/Idea.hs]: Capitalization will be taken care of by css.  Please write like this:

```haskell
"Add creator statement"
```

Comment (fisx) [inline src/Frontend/Page/Idea.hs]: oops, needs translation anyway.  disregard!  (-:

Comment (andorp) [inline src/Frontend/Page/Idea.hs]: @fisx: Could you translate these? :)

**Comment by fisx** [comment]
LGTM.  Please merge if travis is green.

Thanks!

**Pull request merged**